undead-discord.css

```diff
@@ -33,6 +33,66 @@
   border-bottom: 1px solid var(--darkest-bg-color);
 }
 
+/* The following 4 blocks color the keybind-shortcut modal and buttons */
+.ddr-arrows { display: none; }
+
+.keybind-shortcut > span {
+    color: var(--fg-color);
+    background-color: var(--secondary-bg-color) !important;
+    border: none;
+    box-shadow: none;
+    height: 21px;
+    padding: 5px 6px;
+}
+
+.keybind-shortcut > span:active {
+    color: var(--fg-color);
+    border: none;
+    box-shadow: none;
+    height: 21px;
+}
+
+.keybind-shortcut > span > svg > g,
+.keybind-shortcut > span:active > svg > g {
+    fill: var(--fg-color) !important;
+}
+
+/* Colors the help modal (F1 key) */
+.inner-1_1f7b > .modal-content,
+.inner-1_1f7b > .modal-content > .header,
+.inner-1_1f7b > .modal-content > .footer {
+    background-color: var(--bg-color) !important;
+}
+
+.inner-1_1f7b > .modal-content > .form-inner {
+    background-color: var(--secondary-bg-color) !important;
+}
+
+/* Replaces whitish border color with the dark red color */
```

Comment:
What does this comment even mean?

Reply:
Nothing, that was for previous edits, and forgot to remove the comment. It's gone now